Docs: align the documentation and the examples to the new changes (#1181)
metallb/metallb: website/content/installation/_index.md

kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/metallb.yaml
```

{{% notice note %}}

> If you want to deploy metallb using the [experimental frr mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), apply the manifests:

Reviewer: ```suggestion
If you want to deploy MetalLB using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), apply these manifests:
```

```bash
kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/namespace.yaml
kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/metallb-frr.yaml
```

> 

Reviewer: ```suggestion
Please do note that these manifests deploy MetalLB from the main development branch. We highly encourage cloud operators to deploy a stable released version of MetalLB on production environments!
```

Author: Not sure about this. Each tag is pinned to that tag's version

Author: Ah ignore me, this is for  the main branch

namespace: metallb-system

resources:

>   - github.com/metallb/metallb/manifests?ref=v0.9.3

Reviewer: Uff, v0.9.3 is 2 years old now. Should this be updated to just point to `main` and so also be consistent with the change above (FRR manifests)?

```suggestion
  - github.com/metallb/metallb/manifests?ref=v0.9.3
```

Author: I'd say it's better to use the tag as an example.

     - 198.51.100.0/24
```

{{% notice note %}}

> If you want to deploy metallb using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), the following value must be set:

Reviewer: ```suggestion
If you want to deploy MetalLB using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), the following value must be set:
```

```yaml
speaker:
  frr:

>     enabled:true

Reviewer: ```suggestion
    enabled: true
```

```

{{% /notice %}}

## Using the MetalLB Operator

> The MetalLB Operator is available on OperatorHub. It allows configuring MetalLB via CRDs (although [it's planned to have the configuration being handled directly by MetalLB](https://github.com/metallb/metallb/pull/942)).

Reviewer: It might be irrelevant or discouraging to using the MetalLB Operator mentioning that the MetalLB community is discussing (not officially approved yet) a move of CRDs to the MetalLB repository.

```suggestion
The MetalLB Operator is available on [OperatorHub](https://operatorhub.io/operator/metallb-operator). It eases the deployment and life-cycle of MetalLB in a cluster and allows configuring MetalLB via CRDs.
```

{{% notice note %}}

> If you want to deploy metallb using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), you must edit the ClusterServiceVersion resource

Reviewer: ```suggestion
If you want to deploy MetalLB using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), you must edit the ClusterServiceVersion resource
```

named `metallb-operator`:

```bash
kubectl edit csv metallb-operator
```

and change the `BGP_TYPE` environment variable of the `manager` container to `frr`:

```yaml
- name: METALLB_BGP_TYPE
  value: frr
```

> The MetalLB Operator allows to configure MetalLB via CRDs.

Reviewer 2: This line repeats what has been already said in the first sentence of this section:
"The MetalLB Operator is available on OperatorHub. It allows configuring MetalLB via CRDs"

     - 198.51.100.0/24
```

{{% notice note %}}
If you want to deploy MetalLB using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), the following value must be set:

```yaml
speaker:
  frr:
    enabled: true
```

{{% /notice %}}

## Using the MetalLB Operator

> The MetalLB Operator is available on OperatorHub. It eases the deployment and life-cycle of MetalLB in a cluster and allows configuring MetalLB via CRDs.

Reviewer: I see you did not add a link for OperatorHub to https://operatorhub.io/operator/metallb-operator. Is there any reason? As a user, I would appreciate a direct link instead of searching for the OperatorHub website and then finding the MetalLB Operator page.

Author: Done
